gilbreth_support/launch/conveyor_spawner.launch

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0"?>
+<launch>
+  <!-- Load conveyor objects onto parameter server -->
+  <param name="gear_description"
+    command="$(find xacro)/xacro --inorder '$(find gilbreth_support)/urdf/conveyor_objects/gear.xacro'"/>
+
+  <param name="piston_rod_description"
+    command="$(find xacro)/xacro --inorder '$(find gilbreth_support)/urdf/conveyor_objects/piston_rod.xacro'"/>
+
+  <param name="pulley_description"
+    command="$(find xacro)/xacro --inorder '$(find gilbreth_support)/urdf/conveyor_objects/pulley.xacro'"/>
+
+  <param name="gasket_description"
+    command="$(find xacro)/xacro --inorder '$(find gilbreth_support)/urdf/conveyor_objects/gasket.xacro'"/>
+
+  <param name="disk_description"
+    command="$(find xacro)/xacro --inorder '$(find gilbreth_support)/urdf/conveyor_objects/disk.xacro'"/>
+
+  <!-- Start the spawner node -->
+  <node name="conveyor_spawner" pkg="gilbreth_support" type="conveyor_spawner_node" output="screen">
+    <rosparam command="load" file="$(find gilbreth_support)/config/conveyor_objects.yaml"/>
+  </node>
```

Comment:
When we start adding more objects it will be much more manageable to load the urdf descriptions directly into the spawning node using the URDF library.  For now this solution will do, I'll log this nice to have in the issues section.